homeworks/VadimMiroshnik_VadimMir/homework_4/homework4_2.js

```diff
@@ -0,0 +1,103 @@
+function card(){
+    let card = '4561261212345467';
+   
+    let  sum_unpaired = 0, paired_Elemen = 0, unpaired_Elemen = 0, sum_paired = 0, sum = 0, sum_Card;
+    arrayCard = card.match(/[\S\s]{1,1}/g);
```

Comment:
format your code, it is hard to read

Reply:
First and second task?

Reply:
added comments, if in my case this is correct)